.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Question'
+about: 'Questions and requests for support'
+title: 'PLEASE DO NOT USE GITHUB FOR QUESTIONS - CHECK FAQ OR USE FORUM'
+labels:
```

Comment:
Can we directly add "declined: question" here? 🤣

Reply:
I wanted to discuss this 😆 because yes - we can as shown on last screenshot in https://github.com/jacoco/jacoco/pull/948#issuecomment-538687538 🤣however this looks a bit weird given that it will be open for a while. I can imagine GitHub Action that will be closing them automatically, but fear that this might cause that people will be creating multiple ones - anyway this is a bit premature and better be discussed separately later after introduction of clear separation.
 
Also this looks a bit inconsistent with labels `type: bug` and `type: enhancement`. So I'm wondering what do you think about renaming of `declined: question` into just `type: question` for consistency and usage of it here? or without jokes use `declined: question` here?